screen.css

```diff
@@ -1245,3 +1245,572 @@
+  }
+
+  .outer-item li p {
+    color: #253858;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    margin-top: -10px;
+  }
+
+  .desk-about h1 {
+    color: #091e42;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .desk-about li p {
+    color: #344563;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  p.connect {
+    color: #3d4ee4;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .contact-icon {
+    margin-left: 1px;
+    padding: 10px;
+  }
+
+  .contact-icon li a {
+    font-size: 1.5em;
+    color: #505f79;
+  }
+
+  .language {
+    display: flex;
+    text-align: center;
+    margin-top: 120px;
+    justify-content: space-between;
+  }
+
+  .skills {
+    display: flex;
+    justify-content: space-between;
+    text-align: center;
+    padding-top: 20px;
+    border-bottom: 1px solid #acb7c3;
+  }
+
+  .framework {
+    display: flex;
+    text-align: center;
+    justify-content: space-between;
+    border-bottom: 1px solid #acb7c3;
+    padding-top: 20px;
+  }
+
+  .contact {
+    background-color: #6070ff;
+    background-image: url(../images/contactbg.png);
+    background-size: 100%;
+    border-top-left-radius: 100px;
+    margin-top: -100px;
+    padding: 150px 400px 150px 400px;
+  }
+
+  .inner-div {
+    padding-right: 30px;
+    padding-left: 20px;
+    width: 550px;
+    height: 100px;
+    margin-bottom: 100px;
+  }
+
+  .inner-div h2 {
+    color: white;
+    text-align: center;
+    font-size: 40px;
+    font-weight: 400;
+    padding-bottom: 0;
+  }
+
+  #inner-div {
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #ebebff;
+  }
+
+  .form {
+    padding: 20px;
+  }
+
+  li input {
+    background: #fff;
+    border: 1px solid #cfd8dc;
+    font-family: Poppins, sans-serif;
+    border-radius: 8px;
+    width: 100%;
+    height: 50px;
+    padding-left: 10px;
+    color: #000;
+  }
+
+  li textarea {
+    background: #fff;
+    border: 1px solid #cfd8dc;
+    font-family: Poppins, sans-serif;
+    border-radius: 8px;
+    width: 100%;
+    height: 150px;
+    padding-left: 10px;
+    font-size: 15px;
+    line-height: 24px;
+    padding-top: 10px;
+  }
+
+  .inner-list {
+    display: flex;
+    flex-direction: column;
+    margin: auto;
+    width: 100%;
+  }
+
+  .inner-list .flex-item6 button {
+    margin-left: 150px;
+    margin-top: 1px;
+  }
+
+  .all-item:hover {
+    transform: scale(1, 0.8);
+  }
+
+  .desk-item a:hover {
+    color: #3d4ee4;
+  }
+
+  #portfolioPopup {
+    position: fixed;
+    z-index: 1;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+    background-color: rgb(76, 78, 82, 0.7);
+    display: none;
+  }
+
+  ul.works-flex-item.titles-container {
+    margin: 10px;
+  }
+
+  .works-flex {
+    padding: 20px;
+    background: #fff;
+    margin: 50px 100px;
+    border-radius: 10px;
+  }
```

Comment:
Great job @Nowembabazi 🤝

- [ ] You may kindly reduce the size of your image on the desktop popup and also make your whole popup appear on one single screen. You may make sure there are no vertical, and horizontal scrolls.